src/test/resources/zip/xslt-file.xslt

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What does this xslt do?
Does it have to do something complex or should we just have a copy all values xslt?

Reply:
This xslt file is a file I have used before to test if the feature of transformation to edm external works, I don't know what it does exactly in detail. I have changed the name of the file to what it is used for. Do you prefer the name to indicate what this xslt file does?

Reply:
No I think it's fine how you named it now